1-Authentication/5-sign-in-express/README.md

```diff
@@ -1,7 +1,7 @@
 ---
 page_type: sample
-name: A Node.js & Express web app authenticating users against Azure AD for Customers with MSAL Node
-description: This sample demonstrates a Node.js & Express web app authenticating users against Azure Active Directory Customer Identity Access Management (Azure AD for Customers) with Microsoft Authentication Library for Node (MSAL Node)
+name: Sign in users in a sample Node.js & Express web app by using Microsoft Entra External ID for customers
```

Comment:
Isn't the `for customers` going away with the "soon to be confirmed" final names. Just adding this as a check more than anything else